libnmstate/nm/ovs.py

```diff
@@ -175,10 +184,17 @@ def _get_bridge_port_info(port_profile, devices_info):
                 port_name, port_setting, port_slave_names
             )
             port_info.update(port_lag_info)
-        if vlan_mode:
-            port_info["vlan-mode"] = vlan_mode
-            port_info["access-tag"] = port_setting.props.tag
 
+        if vlan_mode:
+            nmstate_vlan_mode = NM_OVS_VLAN_MODE_MAP.get(vlan_mode)
+            if not nmstate_vlan_mode:
+                logging.warnings(
+                    f"OVS Port VLAN mode '{vlan_mode}' is not supported yet"
+                )
+            port_info[OB.Port.VLAN_SUBTREE] = {
+                OB.Port.Vlan.MODE: nmstate_vlan_mode,
```

Comment:
This will be set even for unsupported modes, so it will show here `None`. Is this your intention? Maybe set `UNKNOWN`.

Reply:
My fault. Will fix it.